Makefile

```diff
@@ -9,13 +10,20 @@ JOB_NAME = 1
 
 all: supervisor
 
-IMAGE = "resin/$(ARCH)-supervisor:$(SUPERVISOR_VERSION)"
-SUPERVISOR_IMAGE=$(DEPLOY_REGISTRY)$(IMAGE)
-
 PUBNUB_SUBSCRIBE_KEY = sub-c-bananas
 PUBNUB_PUBLISH_KEY = pub-c-bananas
 MIXPANEL_TOKEN = bananasbananas
 
+ifeq ($(BASE_DISTRO),alpine)
+	IMAGE = "resin/$(ARCH)-supervisor:$(SUPERVISOR_VERSION)-$(BASE_DISTRO)"
+	DOCKERFILE = alpine.$(ARCH)
+else
```

Comment:
My problem is that you can say `make BASE_DISTRO=fedora` or `make BASE_DISTRO=alpien` and it will just silently use debian with no warning or anything, which would be very confusing when you're expecting it to be `fedora` or `alpine` (but typoed)